examples/MLIRGEMM/gemm.mlir

```diff
@@ -0,0 +1,289 @@
+#map0 = affine_map<(d0) -> (d0+10)>
+  
+
+#map10 = affine_map<(d0) -> (d0 * 64)> 
+#map11 = affine_map<(d0) -> (d0 * 64 + 64)>  
+#map12 = affine_map<(d0) -> (d0 * 256)>
+#map13 = affine_map<(d0) -> (d0 * 256 + 256)>  
+#map14 = affine_map<(d0) -> (d0 * 8)> 
+#map15 = affine_map<(d0) -> (d0 * 8 + 8)> 
+#map16 = affine_map<(d0) -> (d0 * 16)> 
+#map17 = affine_map<(d0) -> (d0 * 16 + 16)> 
+
+module{
+
+    func.func private @printMemrefF64(memref<*xf64>)
+
+    // naive muatrix multiplication using linalg dialect 
+    func.func @matmul(%a : memref<?x?xf64>, %b : memref<?x?xf64>, %c : memref<?x?xf64>) {
+      linalg.matmul 
+        ins(%a, %b: memref<?x?xf64>, memref<?x?xf64>)
+       outs(%c:memref<?x?xf64>)
+      return
+    }
+
+    // naive muatrix multiplication using affine dialect 
```

Comment:
maybe change the comment to "naive matrix multiplication **with fix length** using affine dialect" to distinct `matmul1` from `matmul2`?

Reply:
This example is removed.